common/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Is it necessary?

Reply:
.class, .jar etc files are not necessary.  I think you are right and we need to remove it from common/.gitignore file. What do you think?